transport/src/main/java/io/netty/channel/AbstractCoalescingBufferQueue.java

```diff
@@ -30,21 +30,40 @@
 public abstract class AbstractCoalescingBufferQueue {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractCoalescingBufferQueue.class);
     private final ArrayDeque<Object> bufAndListenerPairs;
+    private final PendingBytesTracker tracker;
     private int readableBytes;
 
-    public AbstractCoalescingBufferQueue(int initSize) {
+    /**
+     * Create a new instance.
+     *
+     * @param initSize theinitial size of the underlying queue.
+     */
+    protected AbstractCoalescingBufferQueue(int initSize) {
```

Comment:
this is now unused ... consider removing it ... this class is labeled with `@UnstableApi`.